### DISCUSSION TITLE
似乎有部分地区已经以反诈为借口屏蔽了 Cloudflare CDN 所有 IP 段？

### DISCUSSION THREAD

**Discussion by XIU2** [XIU2/CloudflareSpeedTest#383] [讨论]
最近有用户向我反馈无法使用 CloudflareST，结果最后发现其实该用户的网络有问题，**似乎 Cloudflare CDN  所有 IP 都被屏蔽**。特地问了下，该用户为 **`江苏省苏州市`**，宽带是 **`电信`**、手机是 **`移动`**，均无法访问任何 Cloudflare CDN 的 IP（及其他任何套了的 Cloudflare CDN 的网站都无法访问）。

因为就这一个样本，不清楚是不是 个例 / 其他问题 导致的，有没有其他小伙伴有类似情况的？

**Comment by cxw620** [comment]
https://www.speedtest.net/result/12913320089
ipv6没问题，甚至能走HKIX的线路 速度极快。人在北京，教育网ipv6

**Comment by sz30** [comment]
安徽阜阳 移动宽带似乎也被屏蔽了

**Reply by hsg666888** [reply]
有些网站没有备案运营商就以诈骗或者不安全等借口封掉不让访问真的恶心

**Reply by Davidasx** [reply]
> 有些网站没有备案运营商就以诈骗或者不安全等借口封掉不让访问真的恶心

是的
因为不备案不好抓你，所以干脆封了

**Reply by 1607375424** [reply]
四川成都电信好像也不行了

**Comment by wjr050430** [comment]
我觉得这不大可能。因为我见到所有的被封的cloudflare网站都只是域名污染。估计cf跟gfw有些py交易不封ip

**Comment by sh2288** [comment]
今日发现,不套cdn一切正常,套上cdn就连不上了
换什么IP都是一样
无解中...

**Comment by little-twain** [comment]
证实徐州电信(221.229.173.\*)无法访问，徐州移动(120.195.6.\*)目前正常

**Comment by lucn85** [comment]
苏州电信 套cdn就无法连接，不套正常使用，苏州联通一切正常，所以应该确实是电信做了屏蔽，这个情况出现已经有一个多月了

**Comment by trapan** [comment]
四川，从前天开始的（4月15），没法套cdn了。。。。。

**Comment by Bitcion** [comment]
ipv6应该不好屏蔽吧

**Comment by XIU2** [comment]
@Bitcion 如果真想屏蔽，那都不是事儿，而且 Cloudflare 的 IPv6 都是公开的，直接屏蔽 IP 段不就行啦，虽然 IPv6 数量多，但并不影响屏蔽 IP 段~

**Comment by z836454898** [comment]
> 我觉得这不大可能。因为我见到所有的被封的cloudflare网站都只是域名污染。估计cf跟gfw有些py交易不封ip

应该是怕误伤到一些合法网站吧

**Comment by Ayasenya** [comment]
江苏泰州移动，ipv4可以正常优选，ipv6正常使用过一天，然后就筛选不出来ip了

**Comment by peaceanddemocracy** [comment]
大部分cloudflare的ip段都被干扰，TCP干扰，3分钟，和针对github的干扰方式相同。

**Comment by Dell911** [comment]
郑州的移动宽带也不行了，

**Comment by AkarinLiu** [comment]
赣州的移动宽带ping得通，但是无法正常访问

**Comment by EchoGroot** [comment]
> 四川，从前天开始的（4月15），没法套cdn了。。。。。

成都能用，年前可以使用台湾的节点，80ms左右。年后就不能用台湾的了，现在优选ip基本在200ms左右了。

**Comment by XIU2** [comment]
@EchoGroot 台湾节点是因为公开后，用的人太多，挂了。
https://www.cloudflarestatus.com/
Kaohsiung City - (KHH)   Re-routed

**Comment by bai0012** [comment]
陕西移动，几周前还能测出60，70ms的延迟，2023年开始全部cf服务器延迟都在200ms以上了

**Comment by XIU2** [comment]
@bai0012 那你测出的这些 60~70ms 的 IP，多半就是台湾节点了，前段时间台湾节点用的人太多导致维护了。

**Comment by bai0012** [comment]
> @bai0012 那你测出的这些 60~70ms 的 IP，多半就是台湾节点了，前段时间台湾节点用的人太多导致维护了。

原来是这样啊，谢谢

**Comment by MFWT** [comment]
广东移动，已经测不出HKG的IP了，全是SEA，SJC，甚至LAX，不知道和不和这个有关？

**Comment by XIU2** [comment]
@MFWT 没有关联，单纯是 Cloudflare CDN 调整了路由，这种情况很常见，最近很多亚洲节点（香港、台湾等）因为用的人太多而被调整路由（以此来降低流量成本，亚洲流量单价是欧美的好几倍）。

**Comment by adonis-lsh** [comment]
唉!, 生活在天朝真好

**Comment by mivtec** [comment]
> 广东移动，已经测不出HKG的IP了，全是SEA，SJC，甚至LAX，不知道和不和这个有关？

坐标云南,同CMCC,以前都是走广州再HKG的CMI,虽然调整到SEA,不过带宽还是能跑满. 200M

**Comment by qianbinbin** [comment]
之前家里老人被忽悠拉了个广电宽带，垃圾的一笔，包括 CF 在内的很多境外 IP 都无法访问
江苏电信访问 CF 还可以，但是不时抽风

**Reply by mivtec** [reply]
CF稳定性还是不行,经常会re-route.
换成aws cloudfront舒服多了

**Comment by h1orz** [comment]
电信上周五开始，CF的CDN节点抽风，周日还能通过换CDN节点苟活，今天全封了😢

**Comment by Whirity404** [comment]
广州电信有时候上去的很慢，然后发现，如果是精品香港cn2就会快点，如果不是精品但也是cn2就会非常慢，甚至超时
![Screenshot_20240428_213542](https://github.com/XIU2/CloudflareSpeedTest/assets/123949450/81fbde7c-e6e4-414d-b18f-c8d5f295b459)

**Comment by katherine10131997** [comment]
可以参考騰雲multi CDN
TG https://t.me/katherinechien

**Comment by Davidasx** [comment]
注册的us.kg域名，clouflare dns，安徽移动完全无法访问
有解决办法吗

**Reply by wumingshiali** [reply]
我的一样条件，但广东电信可以

**Reply by Davidasx** [reply]
跟地区有关，江苏一带是最严的。
例如曾经出过VScode官网跳反诈情况。
我的网站是http反诈，https直接被墙……

**Reply by qianbinbin** [reply]
@Davidasx 你的域名我这解析出一个江苏电信的 IP
https://tool.chinaz.com/dns/davidx.us.kg 上面也不是所有都解析到 CF

**Reply by Davidasx** [reply]
经典DNS污染，还有的网站被解析到127.0.0.1

**Reply by qianbinbin** [reply]
.us.kg 反诈重点关照对象，我不挂梯子也会跳转到反诈，买个域名吧

**Reply by Davidasx** [reply]
唉，免费域名被人玩坏了，就只能一网捞反诈了。
本来不是什么大网站，就这样吧，反正也没几个人看。

**Reply by zhengqwe** [reply]
试试[这个](https://github.com/jonm58/ghostcp)
只要ip通就能用

**Comment by Whirity404** [comment]
广东电信根本没墙，就是有时候cloudflare速度比较慢

________________________________
发件人: wumingshiali ***@***.***>
发送时间: 星期四, 十一月 7, 2024 7:59:17 下午
收件人: XIU2/CloudflareSpeedTest ***@***.***>
抄送: Rubedo332 ***@***.***>; Comment ***@***.***>
主题: Re: [XIU2/CloudflareSpeedTest] 似乎有部分地区已经以反诈为借口屏蔽了 Cloudflare CDN 所有 IP 段？ (Discussion #383)


我的一样条件，但广东电信可以

―

**Reply by aeeq** [reply]
自从warp被墙后，SJC 延迟高达 1000ms,LAX还是正常的 160ms

**Comment by Whirity404** [comment]
那eu.org呢？eu.org我记得也是免费域名，但是从来不跳反诈



________________________________
发件人: Aoshi Xu ***@***.***>
发送时间: 星期五, 十一月 8, 2024 5:28:01 下午
收件人: XIU2/CloudflareSpeedTest ***@***.***>
抄送: Rubedo332 ***@***.***>; Comment ***@***.***>
主题: Re: [XIU2/CloudflareSpeedTest] 似乎有部分地区已经以反诈为借口屏蔽了 Cloudflare CDN 所有 IP 段？ (Discussion #383)


唉，免费域名被人玩坏了，就只能一网捞反诈了。
本来不是什么大网站，就这样吧，反正也没几个人看。

―

**Reply by XIU2** [reply]
我记得几年前，就有人遇到过 eu.org 被反诈阻断的情况了，特别是移动更严重。

**Comment by Whirity404** [comment]
可能是eu的申请太长，很少有人用<https://aka.ms/AAb9ysg>
现在基本都不墙的
________________________________

**Comment by Davidasx** [comment]
eu.org好像是80,443被移动阻断了，其他运营商还好
中国移动太可恶了吧，搞墙中墙

**Reply by huangsijun17** [reply]
移动墙中墙，各种拦截。印象里，连360官网都拦了。

**Reply by Davidasx** [reply]
相比于移动，功夫网真的算是善良了
移动直接滥杀，免费域名基本上都（至少在部分地区）不能用了
点名表扬江苏移动，反诈搞的最狠
之前还把vscode官网拦了

**Comment by Whirity404** [comment]
是这样，但是从来没看见cloudflare被拦过
________________________________

**Comment by Limour-dev** [comment]
104.16.0.0/13 段被墙了，心累

**Reply by zhengqwe** [reply]
给🧱上点压力（狗头）
```
hping3 104.16.23.45 --syn -k -p 443 --flood
```
然后7×24小时地跑（狗头）

**Reply by Davidasx** [reply]
> 给🧱上点压力（狗头）
> 
> ```
> hping3 104.16.23.45 --syn -k -p 443 --flood
> ```
> 
> 然后7×24小时地跑（狗头）

6，小心网安直接给你断网（
随便dig找个谷歌ip不就行了

**Reply by ningcol** [reply]
好像是的，我这边也到不开了104.21.48.1

**Comment by konghongdong** [comment]
上海联通也是，直接cf所有ip都ping不通，需要重启猫才能好，不清楚为啥

**Reply by XIU2** [reply]
如果重启光猫就立即恢复了，那么我估计是因为重新拨号导致 IP 变了，从而恢复了。
你下次再遇到可以注意记录一下之前和之后的 IP 是不是变了才恢复的。

**Comment by konghongdong** [comment]
我是路由器拨号的，然后设置了凌晨五点定时重启路由器。前两天我重启了N次路由器都不行，到家把猫重启下就好了。今天路由器又重启了，我发现所有ip又都ping不通了，重启猫还是管用。ip的话每次都会变的

**Reply by XIU2** [reply]
既然每次重新拨号 IP 都会变，但是有时候通，有时候不通，那么你需要记录一下，通的时候宽带 IP 都是哪些，不通的时候又是哪些，多记录几次看看是否有规律。
当然也可能是运营商是通过光猫来区分用户的，所以每次重启光猫都会恢复，但具体我也不了解运营商的相关处理逻辑。

> 另外，你回复时尽量集中在一个帖子楼层中，这样方便管理也方便其他人浏览查看。

**Reply by konghongdong** [reply]
好的，感谢大佬指导，我找找规律。
